build-utils/sentry-instrumentation.ts

```diff
@@ -1,8 +1,12 @@
 /* eslint-env node */
 /* eslint import/no-nodejs-modules:0 */
-const crypto = require('crypto');
-const https = require('https');
-const os = require('os');
+
+import crypto from 'crypto';
+import https from 'https';
+import os from 'os';
+
+import type Sentry from '@sentry/node';
```

Comment:
Unfortunately because we're importing this here, it means `@sentry/node` has to move out of `devDependencies` into `dependencies` otherwise our docker builds will fail since this package isn't available to load.

Reply:
I had the same issue with `@testing-library/*`